rpc/src/lib.rs

```diff
@@ -873,6 +873,6 @@ impl NetApiT for NetApi {
 	}
 
 	fn version(&self) -> Result<String> {
-		Ok("Frontier/v0.1.0".to_string())
+		Ok("43".to_string())
```

Comment:
I think it should be the chain id.

Reply:
Yeah it should be the network ID / chain ID. I got this wrong before.

Reply:
Yea, can you recommend what call that is? I tried reverting to previous value and it wasn't available.

Reply:
Fixed